ansible/roles/etcd/tasks/etcd2-start.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: CoreOS | Enable etcd2
+  service: name=etcd2 enabled=yes
```

Comment:
Added it to the `etcd/defaults/main.yml`, set to true. I don't think it should belong in the top level group_vars. But very easy to overwrite if its useful to optionally skip it.